pkg/liqoctl/install/helm/init.go

```diff
@@ -0,0 +1,47 @@
+package helm
+
+import (
+	helm "github.com/mittwald/go-helm-client"
+	"helm.sh/helm/v3/pkg/repo"
+	"k8s.io/client-go/rest"
+)
+
+// InitializeHelmClientWithRepo initiliazes an helm client for a given *rest.Config and adds the Liqo repository.
+func InitializeHelmClientWithRepo(config *rest.Config) (*helm.HelmClient, error) {
+	opt := &helm.RestConfClientOptions{
+		Options: &helm.Options{
+			Namespace:        LiqoNamespace,
+			RepositoryConfig: liqoHelmConfigPath,
+			RepositoryCache:  liqoHelmCachePath,
+			Debug:            true,
+			Linting:          true,
+			DebugLog:         nil,
+		},
+		RestConfig: config,
+	}
+
+	client, err := helm.NewClientFromRestConf(opt)
+	if err != nil {
+		return nil, err
+	}
+
+	initLiqoRepo(client)
+
+	return client.(*helm.HelmClient), nil
+}
+
+func initLiqoRepo(helmClient helm.Client) {
+	// Define a public chart repository
+	chartRepo := repo.Entry{
+		Name: "liqo",
+		URL:  liqoRepo,
+	}
+
+	if err := helmClient.AddOrUpdateChartRepo(chartRepo); err != nil {
+		panic(err)
```

Comment:
As this will be seen by a user it is probably better to present an error message instead of panic'ing.